site/docs/security/sasl.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Authentication using SASL
+prev: ../tls
+next: ../zookeeper
+---
+
+Bookies support client authentication via SASL. Currently we only support GSSAPI (Kerberos). We will start
+with a general description of how to configure `SASL` for bookies, clients and autorecovery daemons, followed
+by mechanism-specific details and wrap up with some operational details.
+
+## SASL configuration for Bookies
+
+1. Select the mechanisms to enable in the bookies. `GSSAPI` is the only mechanism currently supported by BookKeeper.
```

Comment:
technically we don't treat DIGEST-MD5 as a security feature. there were discussions before to remove `passwd` completely, because it doesn't really make any sense.